scripts/cli/client.yaml

```diff
@@ -0,0 +1,15 @@
+---
+keystore:
+  File: home/.sui/sui_config/sui.keystore
+envs:
+  - alias: localnet
+    rpc: "http://127.0.0.1:9000"
+    ws: ~
+  - alias: devnet
+    rpc: "https://fullnode.devnet.sui.io:443"
+    ws: ~
+  - alias: testnet
+    rpc: "https://fullnode.testnet.sui.io:443"
+    ws: ~
+active_env: testnet
+active_address: "0x14e7ac25259adcc373c96627893976d4fe562a3f3fedce493fc187c5ebd53eee"
```

Comment:
Is there a way we could not include this? Seems odd to have an active address for something that we do not keys for locally

Reply:
Ooouf, I hate it too. Running `sui move build --dump-bytecode-as-base64` requires this `client` file. Per the comment there
```
`sui move build` does not ordinarily require a network connection.
The exception is when --dump-bytecode-as-base64 is specified: In this
case, we should resolve the correct addresses for the respective chain
(e.g., testnet, mainnet) from the Move.lock under automated address management.
```

Reply:
We could change this to allow passing in an `unknown` chain parameter to avoid having the `client.yaml` file. The issue is that then we'd have to wait for the testnet release as that's the binary that we're downloading for running tests.

Reply:
or is there a way to just init it without needing to have a file to copy?

Reply:
Added a flag in the `sui move build` command to ignore this. https://github.com/MystenLabs/sui/pull/20475

Reply:
Generated on the fly and added a issue to later switch to `--ignore-chain` and avoid this thing altogether.